README.md

```diff
@@ -39,19 +39,14 @@ With the ESnext field declarations proposal, the above example can be written as
 class Counter extends HTMLElement {
```

Comment:
@littledan Am I understanding correctly that you'd like to switch to the syntax that @kentcdodds proposed? To replace the bound functions with field declarations? Or has something changed in the proposal since this PR?

Would you like a pull request for that?